++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g Shayepe -f rendezvous.txt -t peer1peer2 -n default -i 7 -d 2
++ dirname ./startserver.sh
++ dirname ./startserver.sh
++ dirname ./startserver.sh
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g PeachUnknown -f rendezvous.txt -t peer1peer2 -n default -i 4 -d 5
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g staysafeandgoodluck -f rendezvous.txt -l -t peer1peer2 -n default -i 1 -d 2
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g Logforme3 -f rendezvous.txt -t peer1peer2 -n default -i 2 -d 1
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g NeelTorRelay2 -f rendezvous.txt -t peer1peer2 -n default -i 6 -d 2
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ BASEDIR=.
+ python3 ./../../rtun-routing/rtun.py -v INFO -g Lule -f rendezvous.txt -t peer1peer2 -n default -i 3 -d 2
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g lw -f rendezvous.txt -t peer1peer2 -n default -i 5 -d 6
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g taipe -f rendezvous.txt -t peer1peer2 -n default -i 8 -d 2
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
/home/agaa/git/imt4905/rtun-routing/lsr.py:565: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if args[3] is 1:
Exception in thread Thread-1549:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1550:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1551:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1552:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1553:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1554:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1558:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1751:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-begonia:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Damnation:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-JimmyBulanik:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1568:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1195:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-985:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-oracle:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-LaserSystem:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-3v1lc0rp666:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-idideditheconfig:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-RNVR217:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1015:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-as212520:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-recodex:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-NTH33R4:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-UenoStation56:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Quintex96:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-CelestialRoom:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1557:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-insula:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-bauruine:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1470:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1352:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-MightyDinoRelay:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-notgoodatmath:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
++ dirname ./startserver.sh
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g staysafeandgoodluck -f rendezvous.txt -l -t peer1peer2 -n default -i 1 -d 2
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g Logforme3 -f rendezvous.txt -t peer1peer2 -n default -i 2 -d 1
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g NeelTorRelay2 -f rendezvous.txt -t peer1peer2 -n default -i 6 -d 2
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g Lule -f rendezvous.txt -t peer1peer2 -n default -i 3 -d 2
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g lw -f rendezvous.txt -t peer1peer2 -n default -i 5 -d 6
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g Shayepe -f rendezvous.txt -t peer1peer2 -n default -i 7 -d 2
++ dirname ./startserver.sh
++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g taipe -f rendezvous.txt -t peer1peer2 -n default -i 8 -d 2
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v INFO -g PeachUnknown -f rendezvous.txt -t peer1peer2 -n default -i 4 -d 5
Exception in thread Thread-883:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-bashfulbear:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-886:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-887:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-hawk:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-NTH20R2:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Lapras:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Unnamed:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-TorDotTeitelNet:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Unnamed:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Assange029us:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Unnamed:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 84, in run
    self.establish_rendezvous(circuit, self.rendezvous_cookie)
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 202, in establish_rendezvous
    circuit._rendezvous_establish(rendezvous_cookie)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 611, in _rendezvous_establish
    cell_established = self.send_relay_wait(inner_cell, CellRelayRendezvousEstablished)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 551, in send_relay_wait
    return w.get()
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvousEstablished
Exception in thread Establish-Unnamed:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-912:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1303:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-xmission1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-PacketPusher:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1598:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Thread-1680:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-artikel10ber10:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-tornode789:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Bates1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1369:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-NTH6R3:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1148:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-Ichotolot63:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-bananaanarchy161:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-4717:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Thread-1587:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/agaa/git/imt4905/rtun-routing/lsr.py", line 657, in shortest_paths
    cost = nx.path_weight(G, v, 'weight')
  File "/usr/local/lib/python3.9/dist-packages/networkx/classes/function.py", line 1288, in path_weight
    raise nx.NetworkXNoPath("path does not exist")
networkx.exception.NetworkXNoPath: path does not exist
Exception in thread Establish-relayon3189:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-fffb:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Quetzalcoatl:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-PancakeWhore:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Snowfall:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-Unnamed:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-k8r:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-t1saczhbu0dgljekrq:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-zinha:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-TheAdopter:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-QuintexAirVPN8:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-thedoctor:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
Exception in thread Establish-IAmMrNimbus:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/agaa/git/imt4905/rtun-routing/rendezvous.py", line 88, in run
    rendezvous2_cell = w.get(timeout=self.timeout)
  File "/home/agaa/git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 134, in get
    raise CellTimeoutError('Timeout wait for ' + cells_format(self._cell_types))
torpy.circuit.CellTimeoutError: Timeout wait for CellRelayRendezvous2
